client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -133,6 +134,13 @@ const PluginsBrowser = ( {
 
 	const selectedSite = useSelector( getSelectedSite );
 	const sitePlan = useSelector( ( state ) => getSitePlan( state, selectedSite?.ID ) );
+	const installedPlugins = useSelector( ( state ) =>
+		getInstalledPlugins( state, [ selectedSite?.ID ] )
+	);
+
+	const activePlugins = installedPlugins.filter(
+		( activePlugin ) => activePlugin.sites[ selectedSite?.ID ].active
+	);
```

Comment:
Suspect something here is broken. Possibly just need `installedPlugins?.filter` to fix.

Click "Browse All" on the popular plugin sections, I get a lot of errors and eventual wsod.

Got to be a bit careful with this browser component as it runs double duty for all these routes / contexts:

```
/plugins (no site)
/plugins/?s=<query>
/plugins/:site
/plugins/:site?s=<query>
/plugins/:category (no site)
/plugins/:category/:site
```